components/Notifications/NewsletterSubscriptions.js

```diff
@@ -6,7 +6,7 @@ import withT from '../../lib/withT'
 import ErrorMessage from '../ErrorMessage'
 
 import FrameBox from '../Frame/Box'
-import { P } from './Elements'
+import { P } from '../Account/Elements'
```

Comment:
I would leave that file in the account folder – newsletter stay separate from notifications for now.